code/game/objects/items/RPD.dm

```diff
@@ -235,6 +235,14 @@ GLOBAL_LIST_INIT(transit_tube_recipes, list(
 	spark_system = null
 	return ..()
 
+/obj/item/pipe_dispenser/equipped(mob/user, slot, initial)
+	. = ..()
+	RegisterSignal(user, COMSIG_MOUSE_SCROLL, .proc/mouse_wheeled)
+
+/obj/item/pipe_dispenser/dropped(mob/user, silent)
+	. = ..()
+	UnregisterSignal(user, COMSIG_MOUSE_SCROLL)
```

Comment:
```suggestion
	UnregisterSignal(user, COMSIG_MOUSE_SCROLL)
	return ..()
```
As a general rule of thumb, use `. = ..()` at the start on initializing events and `return ..()` last in ending events.